.github/workflows/build.yml

```diff
@@ -0,0 +1,77 @@
+name: Build
```

Comment:
We should rename the workflow, the `build` is a subset of stages present in the workflow.
The name of the file should also be updated.